spec/presenters/signup_link_presenter_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
+require "support/taxonomy_helper"
+
+RSpec.describe SignupLinksPresenter do
+  include GovukContentSchemaExamples
+  include TaxonomySpecHelper
+
+  subject(:presenter) { described_class.new(content_item, facets) }
+  let(:facets) { [] }
+  let(:content_item) {
+    content_item_hash = {
+      content_id: 'content_id',
+      base_path: "/mosw-reports",
+      title: 'A finder',
+      name: 'A finder',
+      links: {
+        email_alert_signup: Array.wrap(email_signup_hash)
+      },
+      details: {
+        show_summaries: true,
+        document_noun: "case",
+        sort: [
+          {
+            "name" => "Most viewed",
+            "key" => "-popularity"
+          },
+          {
+              "name" => "Relevance",
+              "key" => "-relevance"
+            },
+          {
+            "name" => "Updated (newest)",
+            "key" => "-public_timestamp",
+            "default" => true
+        }
+        ],
+      }
+    }
+    ContentItem.new(content_item_hash.deep_stringify_keys)
+  }
+
+  let(:email_signup_hash) {
+    {
+      web_url: "http://www.gov.uk/email_signup"
+    }
+  }
+
+  describe 'url helpers' do
+    let(:hidden_facet_hash) {
+      {
+        "filter_key": "hidden",
+        "key": "topic",
+        "type": "hidden",
+        "filterable": true,
+        "allowed_values": [{ "value" => "hidden_facet_content_id" }]
+      }.deep_stringify_keys
+    }
+
+    let(:facets) {
+      [HiddenFacet.new(hidden_facet_hash, facet_values)]
+    }
+
+    describe "#email_alert_signup_url" do
```

Comment:
The test is for `:email_signup_link`, not `#email_alert_signup_url`

Reply:
changed 👍